doc/nrf-bm/migration/nrf5_bm_migration.rst

```diff
@@ -3,298 +3,6 @@
-While some core functionalities remain, they have been implemented differently.
-
-Memory partitioning comparison
-==============================
-
-The following diagram shows the mapping of memory partitions in an nRF5 solution versus the |BMshort| solution.
-
-.. figure:: /images/nrf5_bm_memory_part.svg
-   :alt: nRF5 SDK memory partitioning for DFU
-
-For more details about the two solutions, see `nRF5 SDK DFU memory layout`_ and |BMshort| :ref:`dfu_memory_partitioning`.
-
-Like the nRF5 SDK, the Bare Metal also supports single-bank DFU updates, as well as updates to the SoftDevice and the firmware loader.
-
-Bootloader Changes
-==================
-
-In the nRF5 SDK, the boot process involved two main components: the Master Boot Record (MBR), delivered with the SoftDevice, and the nRF5 Bootloader.
-The MBR served as a simple first-stage bootloader, responsible for jumping to either the application or the bootloader and supporting basic copy functionality for replacing the bootloader or the SoftDevice.
-
-The nRF5 Bootloader was capable of downloading new firmware or combined SoftDevice and Bootloader images, which the MBR could then copy if needed.
-
-In the |BMshort| solution, the bootloader has been replaced by the open-source MCUboot project, which serves as the first-stage (immutable) bootloader.
-MCUboot, also used in the |NCS| and other open-source projects, focuses primarily on image validation.
-It ensures that before an image is started, it has not been tampered with and is signed by the correct author.
-
-Firmware loader changes
-=======================
-
-The firmware loader in |BMshort| is somewhat comparable to the nRF5 Bootloader in functionality.
-However, their roles differ slightly.
-While the nRF5 Bootloader managed both the download process and image validation, the firmware loader in |BMshort| is solely responsible for the download process.
-Image validation is handled by MCUboot.
-
-The process for updating the SoftDevice and the firmware loader is similar in both the old and new solutions, requiring the reuse of application space as storage for the update.
-The key difference lies in how the update is moved from temporary storage to the correct partition.
-In the nRF5 SDK, this copy functionality was managed by the MBR, whereas in |BMshort|, it is handled by an installer image that is downloaded along with the new update.
-
-Protocol Changes
-================
-
-The protocol used to upload new images to the device has also changed.
-The previous nRF5 SDK solution utilized a Nordic proprietary DFU protocol, while |BMshort| adopts the `MCUmgr SMP protocol`_, which is also used in the |NCS|.
-
-Tool Compatibility
-==================
-
-The desktop and mobile tools available for the |NCS| are also compatible with the |BMshort| solution.
-
-Drivers
-*******
-
-For migration of nrfx drivers, see `nrfx migration guides`_.
+`Migration notes - nRF5 SDK to nRF Connect SDK Bare Metal option`_
```

Comment:
I think we should keep what we have. If we see the need to point to the latest as the migration guide is less mature, we could have a note in the top in addition. 

The reason is that when we move forward with the project and make changes to the codebase, the latest migration guide might not match what is in this version. Hence, in the future when we have say v1.2.0 and someone is working on v1.0.0, they will not have a migration guide that corresponds with what they have.  

@bjorn-tore-taraldsen Any comments?